Apps/APN-UIKit/APN UIKit/APN UIKit/View/SettingsViewController.swift

```diff
@@ -108,8 +142,20 @@ func configureSettingsRouter() {
             storage.bgNumOfTasks = bgQMinTasksTextField.text
         }
         
+        // Push enabled
+        if currentSettings.isPushEnabled != pushSwitchState {
+            storage.isPushEnabled = pushSwitchState
+        }
+        
         // Track screen enabled
-//        if currentSettings.isTrackScreenEnabled != trackScreenToggle.va
+        if currentSettings.isTrackScreenEnabled != trackScreenState{
+            storage.isTrackScreenEnabled = trackScreenState
+        }
+        
+        // Debug screen
+        if currentSettings.isDebugModeEnabled != debugModeState {
+            storage.isDebugModeEnabled = debugModeState
+        }
```

Comment:
Similar in my comment in other PRs where I believe the `if` statements add complexity to the code and doesn't add any value. 

I suggest we remove the `if` statements as there doesn't appear to be a need for them.

Reply:
This has been addressed in the [this PR](https://github.com/customerio/customerio-ios/pull/289)